content/docs/sidebar.json

```diff
@@ -552,46 +552,30 @@
     ]
   },
   {
-    "label": "Studio",
     "slug": "studio",
-    "source": "studio/index.md",
     "icon": "studio",
     "children": [
-      {
-        "label": "How It Works",
-        "slug": "overview"
-      },
+      "how-it-works",
       "get-started",
       {
-        "label": "User Guide",
         "slug": "user-guide",
-        "source": "user-guide/index.md",
+        "source": false,
         "children": [
+          "prepare-your-repositories",
           {
-            "label": "Prepare Your Repositories",
-            "slug": "prepare-repositories"
-          },
-          {
-            "label": "Views and Experiments",
-            "slug": "views",
+            "slug": "views-and-experiments",
             "source": false,
             "children": [
               "what-is-a-view",
               "create-view",
+              "configure-a-view",
```

Comment:
> From https://github.com/iterative/dvc.org/pull/3599#pullrequestreview-994168273

- [ ]  Probably we could re-combine some of these pages and put them directly under the Studio UG.

Reply:
> re-combine some of these pages (now buried under /doc/studio/user-guide/views-and-experiments

Moved that part to #3622